vpdq/benchmark/benchmark.py

```diff
@@ -0,0 +1,151 @@
+from numpy import hamming
+import vpdq
+import subprocess
+import os
+import sys
+import argparse
+from pathlib import Path
+from contextlib import contextmanager, nullcontext
+from enum import Enum
+import time
+
+VPDQ_MATCH_DIST = 31
```

Comment:
Nit: We've defined this same constant in many places now

Reply:
There were none in the vPDQ python-binding as it only includes the hashing in the library.